### ISSUE TITLE
side-by-side blank lines style

### ISSUE THREAD

**Issue by stockmind** [rtfpessoa/diff2html#177]
In a side-by-side view (when there are completely new lines or deleted lines in only a side) Is it possible to highlight the blank lines of the other side or append a specific class to target and style them? 
I would like them more visible

Something like this:
https://ibb.co/iH2jg9

At the moment those blank lines are only` <td>` with apparently no content but no specific classes to target them

**Comment by rtfpessoa** [comment]
@stockmind that could be a nice improvement. Should not be hard to implement.
PRs are welcome.

**Comment by stockmind** [comment]
Yeah, probably something like this should work:
https://diffy.org/diff/rk3z7u5a344m6ioxygki6bt9

line 224 of src/side-by-side-printer.js
```javascript
SideBySidePrinter.prototype.generateSingleLineHtml = function(isCombined, type, number, content, possiblePrefix) {
  var lineWithoutPrefix = content;
  var prefix = possiblePrefix;
  var lineClass = 'd2h-code-side-linenumber';
  var contentClass = 'd2h-code-side-line';

  if (!number && !content){
    lineClass += ' d2h-code-side-emptyplaceholder';
    contentClass += ' d2h-code-side-emptyplaceholder';
  }

  if (!prefix) {
    var lineWithPrefix = printerUtils.separatePrefix(isCombined, content);
    prefix = lineWithPrefix.prefix;
    lineWithoutPrefix = lineWithPrefix.line;
  }

  return hoganUtils.render(genericTemplatesPath, 'line',
    {
      type: type,
      lineClass: lineClass,
      contentClass: contentClass,
      prefix: prefix,
      content: lineWithoutPrefix,
      lineNumber: number
    });
};
```

```css
.d2h-code-side-emptyplaceholder {
    background-color: #ddd;
    border-color: #bbb;
}
```

It would just take a bit to me to put up a build environment to try it

**Comment by rtfpessoa** [comment]
That seems like a good way to do it.

**Comment by stockmind** [comment]
Sorry, took me a while. Tested, linted and pushed! 

Pull request: #178

**Comment by rtfpessoa** [comment]
Looking into it right now

**Comment by rtfpessoa** [comment]
Released as version 2.5.0. Thanks for your contribution.
